### ISSUE TITLE
Markdown not getting properly markdowned when there is only one new line in front of it 

### ISSUE THREAD

**Issue by tobiaselindberg** [Uncodin/bypass#198]
A lot of markdown text written after a single line break ends up on previous line not markdown rendered:
E.g numbered lists and bullets. If you write:
Markdown-text below with only one new line:
* bullet 1
* bullet 2

The library would not markdown the above markdown as bullets and "bullet 1" and "bullet 2" would end up on same line as "Markdown-text below with only one new line" while if we write:
Markdown-text below with two new lines:

* bullet 1
* bullet 2

The library would be able to handle this because there is two new lines between "Markdown-text below with two new lines" and the bullets.

Basically this bug report is about that markdown with a single new line in front of the markdown text should also markdowned and the new line should be respected so that the markdown ends up on a new line.

Here is a screen-shot visualizing the issue:
![markdown_linebreak](https://cloud.githubusercontent.com/assets/1694882/12232330/437c1b72-b85e-11e5-82ac-d8a04008547c.png)
